src/sentry/api/endpoints/project_release_files.py

```diff
@@ -46,7 +47,7 @@ def get(self, request, project, version):
 
         file_list = (
             ReleaseFile.objects.filter(release=release).select_related("file").order_by("name")
-        )
+        ).exclude(name=RELEASE_ARCHIVE_FILENAME)
```

Comment:
Nit: It's probably not important, but I think it reads a little better if the exclude is directly before/after the `.filter`